### ISSUE TITLE
T3 Framework looses layout assignment when the plugin is enabled

### ISSUE THREAD

**Issue by davidetaddei** [redCOMPONENT-COM/mvcOverride#8]
Hello, I've found another problem with T3 Framework. 

It seems that when I enable the plugin T3 looses the additional layout assignment and uses only the predefined one.

My configuration is:
- Joomla 3.4.1
- T3 2.4.8
- MVC Override 1.4.10

Can you help?

**Comment by davidetaddei** [comment]
Hello, do you need further details?

**Comment by phproberto** [comment]
Both plugins override core classes to add their own customisations:
https://github.com/t3framework/t3/blob/master/source/plg_system_t3/includes/core/t3.php#L139-L176
https://github.com/redCOMPONENT-COM/mvcOverride/blob/develop/helper/codepool.php#L41-L84

If the T3 plugin is executed before you will lose mvcOverride features and if mvcOverride is executed before you will lose T3 features. 

Both extensions are not compatible.

**Comment by davidetaddei** [comment]
I see, thank you. But is there any way to override classes in T3?

**Comment by phproberto** [comment]
I don't think so. You'd need to develop your custom system plugin to mix the features from both systems so both work as expected. And load it before them.

**Comment by davidetaddei** [comment]
Good to know it :+1:  Thank you for your advices, I will think of it!
